src/views/PeopleSearch/components/PeopleSearchResult/index.js

```diff
@@ -160,7 +203,7 @@ export default class PeopleSearchResult extends Component {
             >
               <Grid item xs={1}>
                 <IMG
-                  className="people-search-avatar"
+                  className={'people-search-avatar'}
```

Comment:
Any particular reason you changed this here? From what I know wouldn't the previous way have worked?

Reply:
Might've been a 'Prettier' modification. I can undo it